src/main/java/org/example/Mapper/Collections/CollectionsStat.java

```diff
@@ -19,21 +19,42 @@ public String getType() {
         return type;
     }
 
+    public void setType(String type) {
+        this.type = type;
+    }
+
     public boolean isInheritance_enabled() {
         return inheritance_enabled;
     }
 
+    public void setInheritance_enabled(boolean inheritance_enabled) {
```

Comment:
Let's open an issue to convert the code to camel case. We can handle that in a separate PR.

Reply:
Issue created.